src/views/Award/AddAward.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { StyledContainer } from '../../components/templates/HomeTemplate.tsx';
+import MainBox from '../../components/atoms/Sections/MainBox.ts';
+import {
+  StyledHeader,
+  StyledHeading,
+} from '../../components/atoms/Heading/Heading.ts';
+import {
+  StyledForm,
+  StyledInput,
+  StyledLabel,
+} from '../../components/atoms/Form/Form.ts';
+import Button from '../../components/atoms/Buttons/Button.ts';
+import useAddAward from '../../api/Award/useAddAward.ts';
+
+import { Award } from './awardTypes.ts';
+
+const AddAward = () => {
+  const { id } = useParams();
+  const [formData, setFormData] = useState<Award>({
+    name: '',
+    value: 0,
+  });
+
+  const navigate = useNavigate();
+
+  const onValueChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const parsedValue = parseInt(event.target.value);
+    if (!isNaN(parsedValue)) {
+      setFormData({ ...formData, value: parsedValue });
+    }
+  };
+
+  const { trigger, isMutating, error } = useAddAward(id);
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const response = await trigger({
+      name: formData.name,
+      value: formData.value,
+    });
+
+    //@todo here we should handle the success state properly
+
+    navigate('/');
```

Comment:
Leszek wrote we should navigate to the list of awards, when the response is succesfull. Can you navigate there and remove the @todo comment when done?

Reply:
Yes, done in[ #c47aa40 commit ](https://github.com/zetonteam/zeton_react/pull/82/commits/c47aa403d6492affc81706ac0bbb2e0925305016)
For this moment, when the awards list view is not prepared, we agreed that after adding the award is to return to the awards tab